hnn_core/network.py

```diff
@@ -401,6 +556,150 @@ def _get_src_type_and_pos(self, gid):
 
         return src_type, src_pos, src_type in self.cellname_list
 
+    def _extract_bias_specs_from_hnn_params(self):
```

Comment:
if you haven't realized yet, one thing I dislike is methods that could be functions ;-) Given the choice between the two, go for a function -- always

Reply:
Yeah, I think this should end up in params.py, wdyt?

Reply:
Yeah, I think this should end up in params.py, wdyt?

Reply:
yeah `params.py` sounds good